apps/comps/rpc/router/competitions/get-agents.ts

```diff
@@ -4,8 +4,14 @@ import { z } from "zod/v4";
 import { ApiError, PagingParamsSchema } from "@recallnet/services/types";
 
 import { base } from "@/rpc/context/base";
+import { cacheMiddleware } from "@/rpc/middleware/cache";
 
 export const getAgents = base
+  .use(
+    cacheMiddleware({
+      revalidate: 20,
+    }),
+  )
```

Comment:
Example of using the middleware when getting competition agents. This is a simple case where we just cache the result for 20 seconds.

Reply:
wow this is so nice and clean, love it